package.json

```diff
@@ -30,11 +30,8 @@
     "@rollup/plugin-json": "4.0.1",
     "babel-jest": "^24.9.0",
     "babel-plugin-transform-charcodes": "^0.2.0",
-    "browserify": "^16.2.3",
```

Comment:
They are moved to `@babel/polyfill` because it is only used when building `dist` of `@babel/polyfill`.